exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -287,6 +287,7 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
   };
 
   return ops$ => {
+    const dispatchEvent = client.debugTarget!.dispatchEvent;
```

Comment:
this will require a rewrite of the babel plugin or use `client.debugTarget!.dispatchEvent` all the time since now this will crash in prod